iceoryx_posh/include/iceoryx_posh/internal/roudi/introspection/mempool_introspection.inl

```diff
@@ -28,7 +28,7 @@ MemPoolIntrospection<MemoryManager, SegmentManager, SenderPort>::MemPoolIntrospe
     , m_senderPort(std::move(senderPort))
     , m_thread(&MemPoolIntrospection<MemoryManager, SegmentManager, SenderPort>::threadMain, this)
 {
-    m_senderPort.activate(); // corresponds to offer
+    m_senderPort.offer();
```

Comment:
at first look m_senderPort confused me :) ,  thinking why still old port type is used

Reply:
Thanks for the reminder! You're right, very confusing. I think @marthtz will adapt this soon, 2nd point on his todo list:

> Clean-up all remaining sender/receiver namings

Reply:
Yes, working on it :)